library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -101,27 +114,26 @@ Http::FilterDataStatus PlatformBridgeFilter::onData(Buffer::Instance& data, bool
   }
 
   envoy_filter_data_status result = on_data(in_data, end_stream, platform_filter_.instance_context);
-  Http::FilterDataStatus status = static_cast<Http::FilterDataStatus>(result.status);
-  switch (status) {
-  case Http::FilterDataStatus::Continue:
+
+  switch (result.status) {
+  case kEnvoyFilterDataStatusContinue:
     if (iteration_state_ == IterationState::Stopped) {
```

Comment:
Instead of having the if and the panic we could just `RELEASE_ASSERT(iteration_state_ != IterationState::Stopped, msg)`

Reply:
Yes, you're absolutely right.